server/venueless/core/services/world.py

```diff
@@ -124,6 +127,8 @@ def _create_room(data, with_channel=False, permission_preset="public", creator=N
     else:
         data["trait_grants"] = {}
 
+    if Room.objects.filter(name=data.get('name')).exists():
```

Comment:
Let's use `__iexact`?

And we probably don't want globally unique names, only world-unique ones.

Reply:
Agreed